.ci_support/linux_64_.yaml

```diff
@@ -1,8 +1,11 @@
 cdt_name:
 - cos6
 channel_sources:
-- conda-forge,defaults
+- conda-forge/label/jupyterlab_server_rc,conda-forge,defaults
```

Comment:
this is the key change to make this a new rc branch

Reply:
Cool!